src/components/BandsContainer.js

```diff
@@ -1,26 +1,20 @@
-import React, { Component } from 'react'
+import React from 'react'
```

Comment:
For this file, I'm being a little picky. It's subjective, but our style guide dictates that the exported component be last in a file with other consts.